.gitignore

```diff
@@ -17,3 +17,5 @@
 /.vscode/**
 /build/**
 *.user
+# Ignore editor backup files
+*~
```

Comment:
Okay,
I will split this pull request.
But first I want to have a collection of commits which pass the build tests.